### ISSUE TITLE
Get scores from inferences from the Text_Classification_Sentiment_Analysis script

### ISSUE THREAD

**Issue by CRochaVox** [NVIDIA/NeMo#7618]
Hello everyone, I trained a model with Text_Classification_Sentiment_Analysis.ipynb and I would like to know if I can extract the scores for each sentiment e.g.:

"I answered a call" is a neutral phrase, what score does the engine understand that this phrase is neutral rather than positive or negative?
Is it possible to obtain this information?
I'm using extraction this way

```
model = nemo_nlp.models.TextClassificationModel.restore_from("path/model")

l_results = model.classifytext(queries=keys_list, batch_size=3, max_seq_length=512)
```

**Comment by RobinDong** [comment]
In the implementation of the function `classifytext()`, it used `torch.argmax()` to directly get the classification label from probability.
```python
                logits = self.forward(
                    input_ids=input_ids.to(device),
                    token_type_ids=input_type_ids.to(device),
                    attention_mask=input_mask.to(device),
                )
    
                preds = tensor2list(torch.argmax(logits, axis=-1))
```
Looks like there is no way to get the "score" in current code.

**Comment by CRochaVox** [comment]
Thank you for the answer, I did what you mentioned and I got it! Thank you very much!
